src/features/Main/components/CardPartner/Images.tsx

```diff
@@ -0,0 +1,8 @@
+const logoPartners: ImageData[] = [
+  { src: 'shipper.png' },
+  { src: 'efishery.png' },
+  { src: 'icehouse.png' },
+  { src: 'learning.png' }
+];
```

Comment:
for better SEO performance, please add a property `alt`  and included that on img tag

additionally, each feature can add a constants folder to put the constant data like this